apps/store/src/features/memberArea/PaymentsSection/PaymentConfig.tsx

```diff
@@ -0,0 +1,75 @@
+import { datadogRum } from '@datadog/browser-rum'
+import styled from '@emotion/styled'
+import { useState } from 'react'
+import { Button, Dialog, theme } from 'ui'
+import { useMemberAreaMemberInfoQuery, useTrustlyInitMutation } from '@/services/apollo/generated'
+import { useAppErrorHandleContext } from '@/services/appErrors/AppErrorContext'
+import { getTrustlyInitMutationVariables } from '@/services/trustly/createTrustlyUrl'
+import { TrustlyIframe } from '@/services/trustly/TrustlyIframe'
+import { useCurrentLocale } from '@/utils/l10n/useCurrentLocale'
+
+type State = { type: 'IDLE' } | { type: 'READY'; trustlyUrl: string }
+
+export const PaymentConfig = ({ startButtonText }: { startButtonText: string }) => {
```

Comment:
nit: just something Malin said about the naming of this. In IEX they talk about it and communicate it as payment connection
```suggestion
export const PaymentConnection = ({ startButtonText }: { startButtonText: string }) => {
```

Reply:
Very good point, I just renamed from `PaymentConfiguration` but that is even better